src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -76,10 +79,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    Message message = new Message(user, text);
-    datastore.storeMessage(message);
+    UrlValidator defaultValidator = new UrlValidator();
```

Comment:
Lets instantiate this closer to where it is used. Its easier to maintain in the future.